bot/commands.js

```diff
@@ -0,0 +1,236 @@
+
+                    break;
+
+                case 1:
+
+                    my100status(msg);
+
+                    break;
+
+                default:
+
+                    help(msg, command, "Shows you a user's activity points and karma from the100.io site.", "[BattleTag]", "[BattleTag] The Battle.net Tag of the user you would like to lookup.");
+            }
+
+            break;
+
+        case (command === userPrefix + "playingnow"): // $playingnow
+
+            playingnow(msg)
+
+            break;
+
+        case (command === userPrefix + 'popcorn'): // $popcorn
+
+            // $popcorn TODO: repalce this with any gif with command $gif 'string'
+
+            popcorn(msg);
+
+            break;
+
+        case (command === userPrefix + 'stats'): // $stats
+
+            // For stats commands the server needs a moment to respond to the request
+            // Here we initially display some placeholder text and when the server provides
+            // its response we replace it with the actual data.
+
+            // TODO: When a user issues the command with no parameter, reference there own BattleTag if in database.
+
+            switch (parameters.length)
+            {
+                case 0:
+
+                    help(msg, command, "Shows you a user's Overwatch quick play statistics.", "[BattleTag]", "[BattleTag] The Battle.net Tag of the user you would like to lookup.");
+
+                    break;
+
+                case 1:
+
+                    msg.reply("Working on your request.")
+                        .then(message =>
+                        {
+                            const msgID = message.id;
+                            stats(msg, msgID);
+                        })
+                        .catch(console.error);
+
+                    break;
+
+                default:
+
+                    help(msg, command, "Shows you a user's Overwatch quick play statistics.", "[BattleTag]", "[BattleTag] The Battle.net Tag of the user you would like to lookup.");
+            }
+
+            break;
+
+        case (command === userPrefix + 'test'): // $test
+
+            // Use this command to execute some code via the "test" command on the test server
+
+            if (bot.user.tag == testbotjoe26Tag)
+            {
+                test(bot, msg);
+            }
+
+            break;
+
+        default: // Command not found
+
+            msg.channel.send('', {
+                embed: {
+                    color: 0xFF0000,
+                    fields: [
+                        {
+                            name: command + ' not found',
+                            value: '```yaml\n' + `Type $commands to display a list of available commands` + '```'
+                        }
+                    ]
+                }
+            });
+    }
+}
```

Comment:
missing ;

Reply:
That's strange. In my source, the semicolon is there on that line.